service/build.gradle

```diff
@@ -80,8 +80,7 @@ task verifyPacts( type: Test ) {
     filter {
         includeTestsMatching "*VerifyPacts*"
     }
-
-}
+ }
```

Comment:
Is this right?

Reply:
I think this was automatic based on the linter/spotless. I didn't manually make this change. Would you like me to revert it?

Reply:
just for my own learning... does that mean that linter or spotless identified the `dockstore-client` directory with the openAPI YAML in it, and added this line as a result?

Reply:
L38 was actually manually added by me. The removal of new line on L81 was automatic by spotless I think.

Reply:
I see - thanks for clarifying